### DISCUSSION TITLE
Location nearby campus.

### DISCUSSION THREAD

**Discussion by HAQ-NAWAZ-MALIK** [community/community#59829] [GitHub Education]
I am doing a course under a university by virtual mode.
I uploaded an official ID card as proof.

Application got rejected several times as it says you need to be nearby campus.

Any solution

**Comment by Akash1134** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @HAQ-NAWAZ-MALIK We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Actions category is a place for our community to discuss and provide feedback on GitHub Actions

Also, you are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction.

I've gone ahead and moved this to the correct category GitHub Education for you. Good luck!

**Comment by samus-aran** [comment]
Hi @HAQ-NAWAZ-MALIK  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by HAQ-NAWAZ-MALIK** [reply]
![Screenshot_2023-08-01-18-45-28-73_e307a3f9df9f380ebaf106e1dc980bb6.jpg](https://github.com/community/community/assets/86514900/c39798af-61b5-497f-a128-cc46009bc035)

Here it is a screen shot, of e-mail i received after rejection.

**Comment by HAQ-NAWAZ-MALIK** [comment]
thank you  so much
